src/templates/build-debug/build-debug-android.ejf

```diff
@@ -15,6 +15,9 @@ const build = {
 name: Build Android Debug App
 
 on:
+  push:
```

Comment:
I would be good to have some info here or in docs why, as it may be confusing when we introduce multi branch

Reply:
I planned to add write about it in docs. A comment here is also a good idea! I will add it

Reply:
added in 048b8ba0232a94a2af1ffad91f49a4e8f9e0c3df